apps/native-component-list/src/screens/AV/AudioPlayer.tsx

```diff
@@ -128,7 +152,74 @@ export default class AudioPlayer extends React.Component<Props, State> {
         setRateAsync={this._setRateAsync}
         setIsMutedAsync={this._setIsMutedAsync}
         setVolume={this._setVolumeAsync}
+        extraIndicator={<JsiAudioBar isPlaying={this.state.isPlaying} sound={this._sound!} />}
       />
     );
   }
 }
+
+// for some reason, iOS returns much smaller sample values
+const inputRange = Platform.OS === 'ios' ? [0, 0.3] : [0, 1];
```

Comment:
Such platform discrepancy worries me a bit 🤔 
Can we ensure we've got the similar ranges on every platform? 🤔

Reply:
That's the plan, but it's not the highest priority right now. I'd like to have both platforms working well first. Then I can experiment with outputs and normalize them some way.

Reply:
Do you track it somewhere? 🤔 
Do you mind creating a Linear project for that? 🤔 
I imagine you can create something like this one: https://linear.app/expo/issue/ENG-745/prepare-and-annotate-modules-code-to-be-the-source-for-generation

Reply:
There's a linear issue for that project, but it needs to be updated 😅 I track everything internally, but thanks for the reminder to sync linear with it 😉